src/cc65/symtab.c

```diff
@@ -821,7 +821,7 @@ SymEntry* AddLocalSym (const char* Name, const Type* T, unsigned Flags, int Offs
 
         /* Set the symbol attributes */
         Entry->Type = TypeDup (T);
-        if ((Flags & SC_AUTO) == SC_AUTO) {
+        if ((Flags & SC_AUTO) == SC_AUTO || (Flags & SC_TYPEMASK) == SC_TYPEDEF) {
```

Comment:
Why should defining a variable with a typedef type mean that it should be put on the C stack?

Reply:
It's not put on the stack by this. The code flow here is exactly how things got done prior to 19bee2befa00bde63f470419383c2982f9e2c45d, as `(SC_TYPEDEF & SC_AUTO) == SC_AUTO` was true back then. There's no harm to clear the `V.Offs` field for `typedef`s.